iina/InitialWindowController.swift

```diff
@@ -64,6 +64,20 @@ fileprivate extension NSColor {
       return NSColor(calibratedRed: 1, green: 0.6, blue: 0.2, alpha: 1)
     }
   }()
+  static let initialWindowNightlyLabel: NSColor = {
+    if #available(macOS 10.14, *) {
+      return NSColor(named: .initialWindowNightlyLabel)!
```

Comment:
I was unable to test Nightly as a run blows up here with a nil.

Reply:
My fault, I unstaged too many changes.